developer/openshift/gitops/argocd/pipeline-service/tekton-results/minio-user-cred-patch.yaml

```diff
@@ -1,22 +0,0 @@
----
```

Comment:
@AndrienkoAleksandr This part seemed like unecessary to me since in tenant config we already have `requestAutoCert: true` which should mount automatically the ca cert. I was able to check that (automatic mounting) in this PR

Reply:
We mount tls certificate to tekton-results controller to make possible do requests to minio tenant without tls error. Without this patch request will fail.

`requestAutoCert: true` mounts certificate to the minio tenant pod, not to the tekton-results controller. So, please, revert this change.

Reply:
You're right I was looking into the wrong pod :) 
Do you know if there is someway ( a param) to set another name than `storage-tls` for the ca secret ?
This way we can have the same name we use in staging/prod for that secret so the config would work for all env

Reply:
I think this name was generated by pattern: "tenant name" + "-" + "tls" . Our tenant name is "storage". So secret name is "storage-tls".

Reply:
Yes I figured that part, however I have not found the param to configure a custom name for the secret

Reply:
Done

Reply:
It's not configurable: https://github.com/minio/operator/blob/master/pkg/apis/minio.min.io/v2/names.go#L64